Logging/composer.json

```diff
@@ -12,8 +12,9 @@
         "yoast/phpunit-polyfills": "^1.0",
         "phpspec/prophecy": "^1.10.3",
         "squizlabs/php_codesniffer": "2.*",
-        "phpdocumentor/reflection": "^3.0||^4.0",
+        "phpdocumentor/reflection": "^3.0||^4.0||^5.3",
         "erusev/parsedown": "^1.6",
+        "fig/log-test": "^1.0",
```

Comment:
Is this an artifact that can be removed? I may have missed it, but couldn't find it being used.

Reply:
Oh, yes! Must have been an orphan from an earlier, then reverted change.

Unfortunately I'm currently on the road, so I'll only be able to apply the removal in a few hours...

Reply:
Never mind this comment, I hadn't realized this would be needed for https://github.com/googleapis/google-cloud-php/blob/077810260b58f5de8a3bbdfd999a5e9a48f71a7f/Logging/tests/Unit/PsrLoggerCompatibilityTest.php to pass when using psr/log:2 since the distributed package doesn't include the test classes like psr/log:1 does. Sorry for the churn, and thanks again for all the work here!